init.go

```diff
@@ -273,6 +278,60 @@ func (a *App) getDefaultDisk() (err error) {
 	return nil
 }
 
+func (a *App) getDefaultCache() (err error) {
+	if a.config.CacheConfig == nil {
+		a.config.CacheConfig = &fs.CacheConfig{}
+	}
+
+	defaultDriverName := a.config.CacheConfig.DefaultCache
+	if defaultDriverName == "" {
+		defaultDriverName = utils.Env("CACHE_DEFAULT_DRIVER", "")
+	}
+
+	cacheDriversConfig := a.config.CacheConfig.CacheDriverConfig
+	if utils.Env("CACHE_DRIVERS") != "" && cacheDriversConfig == nil {
+		if err := json.Unmarshal([]byte(utils.Env("CACHE_DRIVERS")), &cacheDriversConfig); err != nil {
+			return err
+		}
+	}
+
+	// if threre is no driver config, add a default driver
+	if cacheDriversConfig == nil {
+		if defaultDriverName == "" {
+			defaultDriverName = "local"
+		}
```

Comment:
```suggestion
		defaultDriverName := defaultDriverName || "local"
```

Reply:
Should we put `local` into a constant or config?

Reply:
Currently, the code does not use config in many places, so just leave it as it is